src/test/resources/templates/forms.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en"
+      xmlns:th="http://www.thymeleaf.org"
+      xmlns:hx="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <title>Forms Test</title>
+
+    <style>
+        .htmx-indicator {
+            opacity: 0;
+        }
+
+        .htmx-indicator.htmx-request {
+            opacity: 1;
+            animation: ease;
+            animation-duration: 1s;
+        }
+
+        .error {
+            color: red;
+        }
+    </style>
+</head>
+<body>
+<h3>Signup Form</h3>
+<!--/*@thymesVar id="body" type="io.github.wimdeblauwe.hsbt.mvc.FormsController.FormValues"*/-->
+<form hx-post="/forms/newAccount" th:fragment="formBody" th:object="${body}">
+    <div hx-target="this" hx-swap="outerHTML" th:fragment="emailField" th:object="${body}">
```

Comment:
Note the `th:object` on this line and line 28. This is due to the fact that the 2 different fragments can be used independently. I like the simplicity of keeping the fragments together and not duplicating code, so this tradeoff seems worth it, and just something that might need documenting.

Reply:
ok, interesting.